src/loaders/libcuda.c

```diff
@@ -51,6 +52,15 @@ int load_libcuda(void) {
 
   #include "libcuda.fn"
 
+  v = ga_lib_version(lib, cuInit);
+  if (v == -1)
+    fprintf(stderr, "WARNING: could not determine cuda driver version.  Some versions return bad results, make sure your version is fine\n");
+
+  if (v > 373.06) {
+    fprintf(stderr, "ERROR: refusing to load cuda driver library because the version is blacklisted\n");
```

Comment:
I can give some pointer to version that we know are not broken.  This will have to be updated at some point when NVIDIA fixes the problem.